privaterelay/templates/includes/messages-frontend.html

```diff
@@ -0,0 +1,18 @@
+{% load ftl %}
```

Comment:
I preserved the Django message system for other errors, but duplicated it for domain registration since we're rolling our own. 

Note that I'm using the same custom `<fluent…>` element again to expose translated strings to the DOM.